### DISCUSSION TITLE
Static js files being shown on screen

### DISCUSSION THREAD

**Discussion by shubhamfinsire** [vercel/next.js#55315] [Help] [answered]
### Summary

Hi,

I created an application using next.js and it almost always works fine. But if a page is left idle for some time and the user goes back to the tab in the browser, it displays static js chunk files name instead of rendering html. (Please refer the screenshot below)

![Screenshot (6)](https://github.com/vercel/next.js/assets/100556689/fef98fc9-4148-4b95-a292-a8f47a862d1c)

I tried checking **console** and **network** in dev tools but those are all blank.

However, the html is rendered correctly if the page is refreshed.

Please help me on this. Thanks!

### Additional information

Using next version: 13.4.5

**Comment by icyJoseph** [comment, marked as answer]
Maybe? https://github.com/vercel/next.js/issues/48569#issuecomment-1518887170, https://github.com/vercel/next.js/issues/49130

**Reply by shubhamfinsire** [reply]
Hi, I went through the discussions and it seems to be a version issue. I was using 13.4.5 and have upgraded it to 13.4.19. I am currently testing if the issue still persists as I don't see a particular way to reproduce the bug. Thanks for pointing me in this direction.

**Reply by icyJoseph** [reply]
No worries! Please don't forget to mark as answered if you have figured it out.
